core/src/main/resources/error/error-classes.json

```diff
@@ -187,7 +187,60 @@
     "sqlState" : "0A000"
   },
   "UNSUPPORTED_FEATURE" : {
-    "message" : [ "The feature is not supported: <feature>" ],
+    "message" : [ "The feature is not supported: " ],
```

Comment:
do we allow to put parameters in the shared message template?

Reply:
I was considering it, but perhaps that's a "future" improvement.
It requires us to count of the shared prefix, split that off, take the next parameter for the error class and then tail the rest.
It seems error prone.